install_requires.txt

```diff
@@ -0,0 +1,66 @@
+#
```

Comment:
What is the difference between `requirements.txt` and `install_requires.txt`?

Reply:
We discussed this in-person: `install_requires.txt` is used to fill the field of the same name for PyPi. We need this because the `assemble-pip` target doesn't support the flags that we need in the `requirements.txt` file. This is an issue and should be resolved better.

Reply:
https://github.com/vaticle/typedb-ml/issues/162